commcare_connect/workflow/templates/mbw_monitoring/followup_analysis.py

```
@@ -0,0 +1,996 @@
+    return {
+        "pct_duplicate": round(duplicate_count / total * 100) if total > 0 else 0,
+        "mode_value": str(mode_value),
+        "mode_pct": round(mode_count / total * 100) if total > 0 else 0,
+    }
+
+
+def compute_overview_quality_metrics(
+    visit_cases_by_flw: dict[str, list[dict]],
+    mother_metadata: dict[str, dict],
+    parity_by_mother: dict[str, str],
+    current_date: date,
```

Ruff (0.15.1)

[warning] Unused function argument: `current_date`

(ARG001)